### ISSUE TITLE
Couldn't find preset "es2015" relative to directory with a sym link

### ISSUE THREAD

**Issue by lasota** [JeffreyWay/laravel-mix#216]
We are working on a composer package that will will have a publish command.  In that publish command it will move some js and css to the resources folder.  But while working on that package I am just creating a sym link to the packages resources folder.  Without the sym link and having files live in the app/resources/package everything compiles fine.  But if I make app/resources/package a sym link to /package/resources the 'npm run dev' fails.

Error output:
Module build failed: Error: Couldn't find preset "es2015" relative to directory "/path/to/package"

So wondering if there is something that can be done to read the sym links better?  Or maybe I'm not using the best workflow for actively working on package?

node - v7.4.0
npm - 4.0.5

**Comment by lasota** [comment]
node - v7.4.0
npm - 4.0.5

**Comment by lasota** [comment]
`const { mix } = require('laravel-mix');
const path = require('path');
const webpack = require('webpack');

/*
 |--------------------------------------------------------------------------
 | Mix Asset Management
 |--------------------------------------------------------------------------
 |
 | Mix provides a clean, fluent API for defining some Webpack build steps
 | for your Laravel application. By default, we are compiling the Sass
 | file for the application as well as bundling up all the JS files.
 |
 */

mix.webpackConfig({
    plugins: [
        new webpack.ProvidePlugin({
            '_': 'lodash',
            '$': 'jquery',
            'jQuery': 'jquery'
        })
    ],
    resolve: {
        modules: [
            path.resolve(__dirname, 'resources'),
            path.resolve(__dirname, 'node_modules')
        ]
    }
});`

This is my config.  I am using the __dirname but my error notice shows the path to the actual package and not the symlink path.

Module build failed: Error: Couldn't find preset "es2015" relative to directory "path/package/resources/js"

I figure it should be looking for "path/app/resources/js" since its a sym link in that path/app folder.

**Comment by OwenMelbz** [comment]
Possible duplicate? https://github.com/JeffreyWay/laravel-mix/issues/158

**Comment by lasota** [comment]
I think its possible they are related because something funky is going on in that one as well in regards to the path.  Its saying the error is relative to "/Users/owen" but the project and I assume you are running the script are in "/Users/owen/Sites/selesti".

I had looked at that one before and thought it was different because mine works fine without the sym link but it could be similar because it views the root of the script differently.

I did get something to work.

/Users/username/site
/Users/username/package

My setup is just like that but the site has a symlink pulling the package in.  To get it to work I put a package.json file in the "/Users/username/package" folder and only added "yarn install laravel-mix".  Figured that would pull in enough of the dependencies into the node_modules folder.  Everything compiles fine.  I'm guessing thats just a bandaid though.  I would assume the problem comes from node ignoring the sym link and still looking at the actual path on the computer even though I have added ```resolve: {
        modules: [
            path.resolve(__dirname, 'resources'),
            'node_modules'
        ]
    }```
